.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+## PR Description
+
+Please replace this comment with summary, motivation and context of the change.
+List any dependencies required for this change.
+
+You can check the checkboxes below by inserting a 'x' between square brackets
+(without any other characters or spaces) or just check them after publishing PR.
+
+If there is a breaking change, specify dependent PRs in description and
+try to push all related PRs at the same time.
+
+
+## PR Type
+- [ ] Bug fix (change that fixes an issue)
+- [ ] New feature (change that adds new functionality)
+- [ ] Breaking change (has dependencies in other repos or will cause CI to fail)
+
+## PR Checklist
+- [ ] I have folloed code style guidelines
```

Comment:
A few typos:

- (line 19) `followed` + `the`. Should we put a link to the file?
- (line 23) Would it be better to rephrase it to `hard-to-understand parts`?
- (line 27) `testbenches`

Reply:
V2: Implement feedback.